### ISSUE TITLE
Wrong/unknown thermostat running states for Centralite 3156105

### ISSUE THREAD

**Issue by bigun27** [Koenkk/zigbee-herdsman-converters#4284]
I have noticed that Z2M correctly interprets the thermostat's running state in its non-heat pump configuration, but gets it wrong when configured to use a heat pump. Turns out, this is because the thermostat sends completely different values for the running state - some of which Z2M is not set up for. Is this an issue that can be resolved? I have made a table below of all values given by the device, along with the proper function to which they correspond.

| Type     |    |    |
|----------|----|----|
| Cooling  | 22 | 7  |
| Heating  | 9  | 13 |
| Fan only | 4  | 5  |
| Idle     | 0  | 1  |

**Comment by robertalexa** [comment]
These are the current mappings:
https://github.com/Koenkk/zigbee-herdsman-converters/blob/a1b6a7ef34eaf1b337a071a80f2b336986fb29c0/lib/constants.js#L58-L74

From your table, here are some discrepancies I am seeing (but can't make an assessment over which ones are the correct ones)
7 - currently undefined
13 - currently undefined
5 - currently `heat` vs your recommendation of `fan_only`
1 - currently `heat` vs your recommendation of `idle`

Could you double check that your table is correct?

@Koenkk do you have any insights on how to double check what running_state the values belong to?

From the ZCL this is all i can find:
![image](https://user-images.githubusercontent.com/9019306/171032075-63f2bc5f-e72c-4d66-aaaa-709f965aea83.png)

**Comment by bigun27** [comment]
> These are the current mappings:

I saw that file when I was looking around

> From your table, here are some discrepancies

Yes, they are all weird discrepancies, but I can't control what values the device produces.
To be clear, there are other values the device produces that are correct, but I have omitted those values and only included the incorrect ones.

> Could you double check that your table is correct?

It is correct. I don't know why the manufacturer deviated from the standard values half of the time
